# state = -1 is waiting for image
# state =  0 is not have object
# state >= 1 is have a object
int8 state
float64 cx
float64 cy
float64 area